tests/full.test.ts

```diff
@@ -702,6 +702,85 @@ for (let index = 0; index < storages.length; index++) {
       expect(rleaf.entry[0].bigInt()).to.be.eq(1n);
     });
 
+    // https://github.com/iden3/go-merkletree-sql/issues/23
+    it('test issue 23', async () => {
+      const sto = getTreeStorage();
+      const mt = new Merkletree(sto, true, 10);
+
+      await mt.add(1n, 1n);
+      await mt.add(5n, 5n);
+      await mt.add(7n, 7n);
+
+      await mt.delete(7n);
+      await mt.add(7n, 7n);
+    });
+
+    it('test insert delete node right fork', async () => {
```

Comment:
"right branch"?

Reply:
Done. I renamed all unit tests. From fork to branch